web/src/pages/project/AnalysisRunnerView/AnalysisRunnerSummary.tsx

```diff
@@ -99,87 +90,102 @@ const AnalysisRunnerSummary: React.FunctionComponent = () => {
+                                (flatData.filter((log) =>
+                                    filters.every(({ category, value }) =>
+                                        _.get(log, category, '').includes(value)
+                                    )
+                                ).length || 0) / pageLimit
+                            )}
+                            totalSamples={Math.ceil(
+                                flatData.filter((log) =>
+                                    filters.every(({ category, value }) =>
+                                        _.get(log, category, '').includes(value)
+                                    )
+                                ).length
+                            )}
+                            pageNumber={pageNumber}
+                            handleOnClick={handleOnClick}
+                            title={'logs'}
+                        />
+                    </div>
+                    <AnalysisRunnerGrid
+                        data={(!sort.column
+                            ? flatData
+                            : _.orderBy(
+                                  flatData,
+                                  [sort.column],
+                                  sort.direction === 'ascending' ? ['asc'] : ['desc']
+                              )
                         )
-                    )
-                    .slice((pageNumber - 1) * pageLimit, pageNumber * pageLimit)}
-                filters={filters}
-                updateFilter={updateFilter}
-                handleSort={handleSort}
-                sort={sort}
-            />
-            <PageOptions
-                isLoading={loading}
-                totalPageNumbers={Math.ceil(
-                    (flatData.filter((log) =>
-                        filters.every(({ category, value }) => _.get(log, category) === value)
-                    ).length || 0) / pageLimit
-                )}
-                totalSamples={Math.ceil(
-                    flatData.filter((log) =>
-                        filters.every(({ category, value }) => _.get(log, category) === value)
-                    ).length
-                )}
-                pageNumber={pageNumber}
-                handleOnClick={handleOnClick}
-            />
+                            .filter((log) =>
+                                filters.every(({ category, value }) =>
+                                    _.get(log, category, '').includes(value)
+                                )
+                            )
+                            .slice((pageNumber - 1) * pageLimit, pageNumber * pageLimit)}
+                        filters={filters}
+                        updateFilter={updateFilter}
+                        handleSort={handleSort}
+                        sort={sort}
+                    />
+                    <PageOptions
+                        isLoading={loading}
+                        totalPageNumbers={Math.ceil(
```

Comment:
If you have render performance issues, consider splitting the inlined `totalPageNumbers` and `totalSamples ` computations out into a `useMemo`